.gitignore

```diff
@@ -4,7 +4,8 @@ dist/
 venv/
 build/
 docs/source/apiref
-docs/source/examples
+docs/source/release_notes.rst
+docs/source/tutorials
```

Comment:
Why do we ignore these two? Are they auto generated?

Reply:
yes

Reply:
I think we can move auto-generated content to a special directory @pseusys